NOTICE

```diff
@@ -1,5 +1,9 @@
 Apache Dubbo
+<<<<<<< HEAD
+Copyright 2018-2020 The Apache Software Foundation
+=======

```

Comment:
pls fix merge confictions

Reply:
ok，i will do it

Reply:
I am already re-merging the dubbo/master branch in my own branch. Is there anything else I need to do here?